<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>welcomeForm</class>
 <widget class="QWidget" name="welcomeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>801</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));
	color:rgb(255, 255, 255);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="returnToLogin">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 52, 5, 255));

	color: rgb(0, 0, 0);
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
   <widget class="QLabel" name="welcomeLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>401</width>
      <height>76</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>64</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none;</string>
    </property>
    <property name="text">
     <string>WELCOME</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="compareButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>430</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.813, y2:0.880455, stop:0 rgba(255, 170, 255, 255), stop:0.153409 rgba(205, 156, 255, 255), stop:0.852273 rgba(170, 85, 255, 255));
	color: rgb(245, 235, 255);
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>Compare</string>
    </property>
   </widget>
   <widget class="QPushButton" name="keywordSearchButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>360</y>
      <width>181</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.813, y2:0.880455, stop:0 rgba(255, 170, 255, 255), stop:0.153409 rgba(205, 156, 255, 255), stop:0.852273 rgba(170, 85, 255, 255));
	color: rgb(245, 235, 255);
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>search keyword</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stockSearchButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>360</y>
      <width>181</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.813, y2:0.880455, stop:0 rgba(255, 170, 255, 255), stop:0.153409 rgba(205, 156, 255, 255), stop:0.852273 rgba(170, 85, 255, 255));
	color: rgb(245, 235, 255);
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>search stock</string>
    </property>
   </widget>
   <widget class="QLabel" name="keywordLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>120</y>
      <width>251</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none;</string>
    </property>
    <property name="text">
     <string>Keywords</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="keywordsTable">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>170</y>
      <width>311</width>
      <height>185</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: black;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <column>
     <property name="text">
      <string>Word</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Searches</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="stocksLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>151</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none;</string>
    </property>
    <property name="text">
     <string>Stocks</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="stocksTable">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>170</y>
      <width>311</width>
      <height>185</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: black;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Ticker</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Price ($)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Change ($)</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>110</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.813, y2:0.880455, stop:0 rgba(255, 170, 255, 255), stop:0.153409 rgba(205, 156, 255, 255), stop:0.852273 rgba(170, 85, 255, 255));
	color: rgb(245, 235, 255);
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
